DatadogSessionReplay/Tests/SessionReplayTests.swift

```diff
@@ -17,12 +17,14 @@ class SessionReplayTests: XCTestCase {
 
     override func setUpWithError() throws {
         core = FeatureRegistrationCoreMock()
+        CoreRegistry.register(default: core)
```

Comment:
/question is it necessary for this test?

Reply:
Yes, I need it for the new test, otherwise there are no instance registered in `CoreRegistry` and `isFeatureEnabled` will always return `false`. I can also include it only where it's needed, in `testWhenEnabledMultipleTimes_itPrintsError`. Or remove the test altogether.

Reply:
ah sorry, I missed the test below. Let's keep it as is 👍